cookbooks/cdo-apps/libraries/cdo_apps.rb

```diff
@@ -17,7 +17,9 @@ def setup_app(app_name)
     execute "setup-#{app_name}" do
       command "bundle exec rake #{app_name}:setup_db --trace"
       cwd app_root
-      environment env.merge(node['cdo-apps']['bundle_env'])
+      environment env.merge(node['cdo-apps']['bundle_env'], {
+        HOME: "/home/#{user}"
+      })
```

Comment:
This chef recipe executes `bundle exec rake dashboard:setup_db`, which in turn loads pycall.rb, which does a `pdm run which python` to find the venv. However, despite running the execute with `user user`, chef does NOT set the env vars so its like running with `su blah` rather than `su - blah`. PDM reads the `HOME` env var to decide where to store config, and finds it cannot read `/root/.config` and bails out.

The fix? Just set HOME before shelling out to run `bundle exec rake dashboard:setup_db`